isis/src/base/objs/Preference/TestPreferences

```diff
@@ -155,42 +155,42 @@ EndGroup
 ########################################################
 
 Group = DataDirectory
```

Comment:
This file need to be deleted. It was moved to the isis directory alongside IsisPreferences. If something is using it we need to fix the something

Reply:
Some unit test was using it. I've added this as a task on our document.